src/ucp/wireup/wireup_cm.c

```diff
@@ -118,6 +118,18 @@ static int ucp_cm_client_try_fallback_cms(ucp_ep_h ep)
     return 1;
 }
 
+static ucp_rsc_index_t
+ucp_cm_ep_tl_bitmap_get_dev_idx(ucp_worker_h worker, uint64_t tl_bitmap)
```

Comment:
func name is not accurate and don't conform with code style

Reply:
also, IMO this func is err prove because can be called from limited context (need to be sure that tl_bitmap contains TLs from singe device)

Reply:
added checks